op-chain-ops/cmd/check-delta/aggregate-channels.sh

```diff
@@ -33,6 +33,7 @@ singular_batch_count=0
 
 channels_with_invalid_batches=()
 
+# shellcheck disable=SC2016
 batch_type_counter_jq_script='reduce .batch_types[] as $batch_type (
     {"span_batch_count": 0, "singular_batch_count": 0};
     if $batch_type == 1 then
```

Comment:
[SC2016](https://www.shellcheck.net/wiki/SC2016) is "Expressions don't expand in single quotes, use double quotes for that". Seems we're generating a jq query here and don't want expansion, so I disabled the check here